README.md

```diff
@@ -5,9 +5,21 @@
 This is the main source of the
 [website for the awesome window manager](https://awesomewm.org/).
 
-The website is based on [ikiwiki](https://ikiwiki.info/). You can build the web
-page locally by running `make`. The result will be in `html/`. Besides ikiwiki,
-you will also need [PerlMagick](https://www.imagemagick.org/script/perl-magick.php).
+## Requirements
+
+- [ikiwiki](https://ikiwiki.info/)
+- [PerlMagick](https://www.imagemagick.org/script/perl-magick.php) (optional,
+  for images)
+
+## Hacking
+
+You can build the web page locally by running `make`, which will generate the
+output in `html/`, where `html/index.html` is the start page.
+
+You can run a simple webserver using Python:
+
+    $ cd html
+    $ python3 -m http.server -b localhost 8000 &
```

Comment:
It's better, since we are using a 'trick' to make it works. It's smoother using the python webserver.